quipucords/api/scanjob/tests_scanjob.py

```diff
@@ -259,7 +264,10 @@ def test_create_invalid_srcs_id(self):
     @patch('api.scanjob.view.start_scan', side_effect=dummy_start)
     def test_create_default_host_type(self, start_scan):
         """A valid create request should succeed with defaulted type."""
-        data = {'sources': [self.source.id]}
+        data = {'sources': [self.source.id],
+                'options': {'disable_optional_products': {'jboss_eap': True,
```

Comment:
should have a create with a test of 'disable_optional_products' when its not a JSON object/python dictionary. Could be a string, array list, etc.